src/app/app.component.ts

```diff
@@ -1,10 +1,14 @@
 import { Component } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.less']
 })
 export class AppComponent {
-
+  language = navigator.language.split(/[-_]/)[0];
```

Comment:
ngx-translate has methods to determinate the language already. In some browsers like IE or safari this navigator.language is an array so probably we should use that methods instead of this technique. 

```typescript
const browserLanguage = translateService.getBrowserLang();
translateService.setDefaultLang('en'); //So that it loads english language if the browser doesn't have a defined one.
translateService.use(browserLanguage); 
```